modules/monitoring-removing-user-defined-alerting-rules.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * monitoring/managing-alerts.adoc
+
+:_content-type: PROCEDURE
+[id="removing-user-defined-alerting-rules_{context}"]
+= Removing user-defined alerting rules
+
+As a cluster administrator, you can remove custom, user-defined alerting rules for core {product-title} platform monitoring.
+
+.Prerequisites
+
+* You are logged in as a user that has the `cluster-admin` role.
+* You have installed the OpenShift CLI (`oc`).
+
```

Comment:
same as https://github.com/openshift/openshift-docs/pull/48754/files#r939802522
we should add
`You have enabled TechPreview feature and wait for all the nodes become ready`

Reply:
Added a line to the prerequsites per your feedback.